app/src/main/java/fr/free/nrw/commons/wikidata/model/WikidataSetClaim.kt

```diff
@@ -0,0 +1,30 @@
+package fr.free.nrw.commons.wikidata.model
+
+import com.google.gson.annotations.SerializedName
+import fr.free.nrw.commons.wikidata.WikidataProperties
+import org.wikipedia.wikidata.Snak_partial
+
+data class WikidataSetClaim(
```

Comment:
This is a `Statement_partial` though not as partial

Reply:
Thanks for pointing me to `Statement_partial`. I have updated the PR.